content/show/coder-radio/subscribe.md

```diff
@@ -1,6 +1,21 @@
 ---
 title: "Subscribe"
-date: 2022-08-16T05:32:13-04:00
+date: 2022-08-25
 draft: false
+rss: "https://coder.show/rss"
+direct_links:
+    apple: https://podcasts.apple.com/us/podcast/coder-radio/id534941512
+    google_podcast: https://podcasts.google.com/?feed=aHR0cHM6Ly9jb2Rlci5zaG93L3Jzcw==
+    castro: https://castro.fm/itunes/534941512
+    castbox: https://castbox.fm/channel/Coder-Radio-Video-id2482150
```

Comment:
I can't play any Coder Radio episode on the castbox page.

Reply:
Nice catch that is the old video feed, my bad... pushing fix now. Standby.

Reply:
The new feed isn't on Castbox it only has the old OGG and Video feeds on Castbox... if you look at https://coder.show/ site the link for castbox doesn't even link to Castbox it links to radio republic.

Reply:
So maybe it makes sense to remove the castbox link until this is fixed?

Reply:
makes sense to me

Reply:
very similar issue here aswell, and could use the same treatment if the symptoms are similar (i.e. ScaleEngine)

https://github.com/JupiterBroadcasting/network/issues/4

can someone confirm?

Reply:
I haven't been able to look at this yet, but I actually used the castbox app exclusively up till I've recently been starting to use podverse a bit. The specific episode that @reclaimingmytime linked to works perfectly fine in the app for me.

Give me till later tonight EST, and I'll see if I can figure out what's going on 😅.

Reply:
created related issues:

* #347
* https://github.com/JupiterBroadcasting/network/issues/7